src/StatusQ/Components/StatusMessage.qml

```diff
@@ -39,9 +37,26 @@ Rectangle {
     property bool isAppWindowActive: false
     property bool editMode: false
     property bool isAReply: false
+
+    property bool hasMention: false
+    property bool isPinned: false
+    property string pinnedBy: ""
+    property bool hasExpired: false
+    property string timestamp: ""
+
     property StatusMessageDetails messageDetails: StatusMessageDetails {}
     property StatusMessageDetails replyDetails: StatusMessageDetails {}
 
+    property var timestampToString: (value) => {
+                                        const parsed = parseInt(value, 10);
+                                        return Qt.formatTime(new Date(parsed), "hh:mm");
```

Comment:
This should respect the user's locale, see https://doc.qt.io/qt-5/qml-qtqml-date.html#toLocaleTimeString-method